### PR TITLE
Record softfail for CC tests known issues

### PR DESCRIPTION
**Pull request by Amrysliu**
There are some known issues in CC testing. In case not block the
following tests, we need to record soft failure for them.

Relate: https://progress.opensuse.org/issues/104862
Verify run: https://openqa.suse.de/tests/7968051#

### CODE REVIEW

File: lib/audit_test.pm [hunk 1]
```diff
@@ -158,14 +157,23 @@ sub compare_run_log {
...
         $flag = 'softfail' if ($flag ne 'fail');
     }
     else {
-        if ($name eq 'ssh04' && is_s390x) {
-            record_soft_failure('Test case ssh04 fails in s390x is a known issue, see poo#99096');
-            return 'softfail';
+        my $arch = get_var('ARCH');
+        if ($softfail_tests->{$arch}) {
+            if (my $msg = $softfail_tests->{$arch}->{$name}) {
+                record_soft_failure($msg);
```
Comment (lilyeyes) [inline lib/audit_test.pm:174]: Check https://openqa.suse.de/tests/7968051#step/ip_eb_tables/36:
Could you please add/show the test case name?

Comment (jouyingbin) [reply lib/audit_test.pm:174]: same as lily's comment, it looks like it shows only.
```
# Soft Failure:
bsc#1192324
```

**Review by lilyeyes — APPROVED** [review]
lgtm

**Review by rfan1 — APPROVED** [review]
LGTM

**Review by jouyingbin — APPROVED** [review]
LGTM

**Pull request merged**